docs/en/01_Configuration.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Configuring TinyMCE
+icon: code
+---
+
+# Configuring TinyMCE
+
+## Adding and removing capabilities
+
+In its simplest form, TinyMCE configuration includes adding and removing buttons and plugins.
+
+You can add plugins to the editor using the [`TinyMCEConfig::enablePlugins()`](api:SilverStripe\TinyMCE\TinyMCEConfig::enablePlugins()) method. This will
+transparently generate the relevant underlying TinyMCE code.
+
+> [!TIP]
+> We've done an explicit `instanceof` check here for correctness, but in reality unless your project also uses an alternative WYSIWYG editor, you can safely omit that check. The remaining examples in this documentation will omit the check.
+
+```php
+// app/_config.php
+use SilverStripe\Forms\HTMLEditor\HTMLEditorConfig;
+use SilverStripe\TinyMCE\TinyMCEConfig;
+
+$editorConfig = HTMLEditorConfig::get('cms');
+if ($editorConfig instanceof TinyMCEConfig) {
+    $editorConfig->enablePlugins('emoticons');
+}
+```
+
+> [!NOTE]
+> This utilities the TinyMCE's [`external_plugins`](https://www.tiny.cloud/docs/tinymce/6/editor-important-options/#external_plugins)
+> option under the hood.
+
+Plugins and advanced themes can provide additional buttons that can be added (or removed) through the
+configuration. Here is an example of adding a `ssmacron` button after the `charmap` button:
+
+```php
+// app/_config.php
+use SilverStripe\TinyMCE\TinyMCEConfig;
+TinyMCEConfig::get('cms')->insertButtonsAfter('charmap', 'ssmacron');
```

Comment:
```suggestion
use SilverStripe\TinyMCE\TinyMCEConfig;

TinyMCEConfig::get('cms')->insertButtonsAfter('charmap', 'ssmacron');
```

Reply:
We don't have a standard for this - some docs always have the gap, some omit it when there's very little code.
I'll add the gap to avoid unnecessary ping pong.